src/lib/utils/sqlite_wrapper.cpp

```diff
@@ -226,8 +225,8 @@ std::shared_ptr<Table> SQLiteWrapper::Connection::execute_query(const std::strin
   return result_table;
 }
 
-void SQLiteWrapper::Connection::raw_execute_query(const std::string& sql, const bool allow_failure) const {
-  char* err_msg;
+void SQLiteWrapper::Connection::raw_execute_query(const std::string& sql) const {
+  char* err_msg = nullptr;
```

Comment:
I think `auto way` of initializing `err_msg` would be something `auto* err_msg = std::static_cast<char*>(nullptr);`. I don't see that this is any better here. But I don't mind changing it.

Reply:
What do you think @dey4ss?

Reply:
😱
Don't know if we can do sth with `new` (like [here](https://rules.sonarsource.com/cpp/RSPEC-4334)) with the primitive (and even then, we have `new`...). Just keep it as it is, I guess